contrib/hikari/src/main/scala/doobie/contrib/hikari/HikariTransactor.scala

```diff
@@ -28,6 +27,10 @@ object hikaritransactor {
     def initial[M[_]: Monad : Catchable: Capture]: M[HikariTransactor[M]] = 
       Capture[M].apply(new HikariTransactor(new HikariDataSource))
 
+    /** Constructs a program that yields a `HikariTransactor` from an existing `HikariDatasource`. */
+    def apply[M[_]: Monad : Catchable: Capture](hikariDataSource : HikariDataSource): M[HikariTransactor[M]] = 
+      Capture[M].apply(new HikariTransactor(hikariDataSource))
```

Comment:
This doesn't need to be captured in `M` because it's not a side-effect; you can just return `HikariTransactor[M]`.